.github/workflows/ci.yml

```diff
@@ -249,3 +249,45 @@ jobs:
         dotnet add csharp.test package ParquetSharp -v ${{ steps.get-version.outputs.version }}
     - name: Build & Run .NET unit tests
       run: dotnet test csharp.test --configuration=Release --framework ${{ steps.dotnet.outputs.framework }}
+    outputs:
+      version: ${{ steps.get-version.outputs.version }}
```

Comment:
Is this used? I was expecting to see this version used in the publish-release action but it looks like that computes the version itself

Reply:
No, it's not used 😅
I was planning to use it to avoid recomputing it below, but decided against it because I am planning a future refactor where `publish-release` won't have a direct dependency on `test-nuget`.

Thanks a lot for spotting it!